api/common/cloudspec/cloudspec.go

```diff
@@ -16,20 +16,21 @@ import (
 // CloudSpecAPI provides common client-side API functions
 // to call into apiserver/common/cloudspec.CloudSpec.
 type CloudSpecAPI struct {
-	facade base.FacadeCaller
+	facade   base.FacadeCaller
+	modelTag names.ModelTag
 }
 
 // NewCloudSpecAPI creates a CloudSpecAPI using the provided
 // FacadeCaller.
-func NewCloudSpecAPI(facade base.FacadeCaller) *CloudSpecAPI {
-	return &CloudSpecAPI{facade}
+func NewCloudSpecAPI(facade base.FacadeCaller, modelTag names.ModelTag) *CloudSpecAPI {
```

Comment:
yeah I know. it needs a model tag and the caller already has a model tag so passing a model tag seemed the most pragmatic approach.